permissions/Phone.permission

```diff
@@ -49,3 +49,7 @@ dbus-user.broadcast com.jolla.PinQuery=com.jolla.PinQuery.*@/*
 # BEG sessionbus-com.jolla.settings
 dbus-user.call com.jolla.settings=com.jolla.settings.ui.*@/*
 # END sessionbus-com.jolla.settings
+
+# Call recordings
```

Comment:
Maybe this should be permission of it's own: "CallRecordings"

Data location can be changed later.

Reply:
:+1: